### DISCUSSION TITLE
How to write a <![CDATA[]]> section using ol.format.WFS.writeTransaction ?

### DISCUSSION THREAD

**Discussion by Mysterius** [openlayers/openlayers#15592] [Q&A]
Hello,

I've been trying to have Geoserver recognize the line breaks in the WFS transactions generated by Openlayers 8.1.0.
According to [this Geoserver issue](https://osgeo-org.atlassian.net/browse/GEOS-8227), the solution is to wrap the value of the parameter with `<![CDATA[]]>`.

However, I can't find anything in the writeTransaction options that could let me specify which parameters should be wrapped with CDATA.

The problem was already pointed by someone else in #10127 

I'm at a loss here ^^'.

**Comment by ahocevar** [comment]
The patch in #10127 looks like a good fix. Do you want to create a pull request with that fix?

**Reply by Mysterius** [reply]
I would but I'm a complete noob in everything related to coding and have never used github in any other way than interact with people ^^'. I don't even know how this works so I may not be the best person for the job !

**Comment by MelkorCC** [comment]
Fixed in #16596
